torch/_functorch/_aot_autograd/jit_compile_runtime_wrappers.py

```diff
@@ -309,6 +315,23 @@ def aot_dispatch_autograd(
             if not hasattr(compiled_fw_func, "_boxed_call"):
                 compiled_fw_func = make_boxed_func(compiled_fw_func)
 
+            if tracing_context and tracing_context.fakify_first_call:
+                out = [n.meta["val"] for n in (list(fw_module.graph.nodes)[-1].args[0])]
+                # will only be set for inductor
+                if fwd_output_strides:
+                    for i in range(len(out)):
+                        if not isinstance(out[i], Tensor):
+                            continue
+                        if all(
+                            guard_size_oblivious(s1 == s2)
```

Comment:
🤔 I guess we're effectively specializing (and adding guards) on whether or not inductor generated different output strides. Are you worried this will add a bunch of extra guards though, if we have a graph with 100+ outputs?

Reply:
I dont want to add any extra guards here.. Let me switch to `suppress_guards`